lib/generator/services/Parse_json_cities.rb

```diff
@@ -1,17 +1,20 @@
 require 'json'
-
+# generator for ID, requires testing as well
 class ParseJson
   attr_accessor :json_data
 
   def initialize
     @json_data = JSON.parse(File.read('city_list.json'))
   end
-
+  
+  # method to search through json file and return city id
   def get_id_by_name(city_name, country)
```

Comment:
This is a nice method but we shouldn't have to give the program the city name and country to get back the ID. We want to retrieve a random city id.